Mobile Buy SDK/Mobile Buy SDK/Models/BUYCartLineItem.m

```diff
@@ -61,4 +61,25 @@ - (NSUInteger)hash
 	return hash;
 }
 
+#pragma mark - BUYPersistence
+
++ (instancetype)instanceWithPlistDictionary:(NSDictionary *)dictionary
+{
+	BUYProductVariant *variant = [BUYProductVariant instanceWithPlistDictionary:dictionary[@"variant"]];
```

Comment:
You should probably not serialize the entire variant object in the cart. The variant details might change between persisting the cart and loading it. Store the variant's `identifier` instead.